app/src/main/java/com/ayush/weatherapp/home/HomeActivity.java

```diff
@@ -276,4 +286,13 @@ private void fetchHomeDetails() {
             new Intent(Settings.ACTION_LOCATION_SOURCE_SETTINGS)));
     dialog.setCancelable(false).show();
   }
+
+  @Override public boolean onQueryTextSubmit(String query) {
+    presenter.fetchNewLocation(query);
+    return false;
```

Comment:
Should this be true? Mention the reason for yes or no in comment.

Reply:
:ok: